documentation/modules/appendix_crds.adoc

```diff
@@ -1285,6 +1287,67 @@ Configuration of how TLS certificates are used within the cluster. This applies
 |string (one of [replace-key, renew-certificate])
 |====
 
+[id='type-JmxTransSpec-{context}']
+### `JmxTransSpec` schema reference
+
+Used in: xref:type-KafkaSpec-{context}[`KafkaSpec`]
+
+
+[options="header"]
+|====
+|Property                  |Description
+|image              1.2+<.<|The image to use for the JmxTrans.
+|string
+|outputDefinitions  1.2+<.<|Defines the output hosts that will be referenced later on. For more information on these properties see, xref:type-JmxTransSpec-reference[`JmxTransSpec` schema reference].
+|xref:type-JmxTransOutputDefinitionTemplate-{context}[`JmxTransOutputDefinitionTemplate`] array
+|kafkaQueries       1.2+<.<|Queries to send to the Kafka brokers to define what data should be read from each broker. For more information on these properties see, xref:type-JmxTransSpec-reference[`JmxTransSpec` schema reference].
```

Comment:
@PaulRMellor See here the circular linking. I don't think this makes sense. Is this really want you intended?